### DISCUSSION TITLE
Broadcasting

### DISCUSSION THREAD

**Discussion by ProdigyView** [edumeet/edumeet#848] [Ideas]
Is it possible to broadcast? Where x participants will have videos that is broadcasted to Y amount of participants? Could the MediaSoup API be leveraged to achieve this?

**Comment by misi** [comment]
You can do something similar to broadcasting, just mute audio and video of the participants and has one presenter.
The real broadcasting/webcaset is on our long term roadmap.

**Comment by ProdigyView** [comment]
@misi Thanks! Is there a broadcasting alpha or beta I can try out?

**Comment by so010** [comment]
> Thanks! Is there a broadcasting alpha or beta I can try out?

No not yet! And yes mediasoup could deliver this easily. If you feel so just begin with coding - I can point you in right direction.
